Documentation/ExtensionArchitecture/FileStructure/Configuration/TCA/Index.rst

```diff
@@ -14,6 +14,14 @@ The folder :file:`EXT:my_extension/Configuration/TCA/` may contain or override
 All files in this directory are automatically included during the TYPO3
 :ref:`bootstrap <bootstrapping>`.
 
+..  versionadded:: 12.0
+    Files within :file:`Configuration/TCA/` files are loaded within a dedicated scope.
```

Comment:
```suggestion
    Files within :file:`Configuration/TCA/Overrides/` are loaded within a dedicated scope within TYPO3.
```

Reply:
The whole TCA directory (with Overrides subdirectory) is targetted by the described behavior.
Should I mention both directories?

Reply:
In the TCA directory (without Overrides) you are returning an array, so there should be no leakage at all. But maybe someone defines variables there using then in the TCA definition. But that would be an edge case. So, would be okay. Thanks.